### PR TITLE
Add an explanatory sentence to the 'compositing' material model.

### PR DESCRIPTION
**Pull request by bangerth**
Following up on a conversation I had with @paddyball today.

/rebuild

### CODE REVIEW

File: source/material_model/compositing.cc [hunk 1]
```diff
@@ -246,7 +246,12 @@ namespace aspect
                                    "compositing",
                                    "The ``compositing'' Material model selects material model properties from a "
                                    "given set of other material models, and is intended to make mixing different "
-                                   "material models easier."
+                                   "material models easier. This is useful, for example, when wanting to "
+                                   "use the melting parameterization of the `melt simple' model (which has "
```
Comment (anne-glerum) [inline source/material_model/compositing.cc:250]: In the sentence above, two ticks are used for the name of the compositing material model (``compositing''). I don't know whether we use one or two ticks, but I guess it should be the same everywhere.

File: source/material_model/compositing.cc [hunk 2]
```diff
@@ -246,7 +246,12 @@ namespace aspect
...
+                                   "a relatively simple viscosity model that only allows for a "
+                                   "temperature- but not strain rate-dependent viscosity) with a more "
+                                   "realistic viscosity model such as that provided by the "
+                                   "`diffusion dislocation' model."
```
Comment (anne-glerum) [inline source/material_model/compositing.cc:254]: same here

**Review by anne-glerum — CHANGES_REQUESTED** [review]

**Comment by bangerth** [comment]
Good points, @anne-glerum . Fixed!

**Review by anne-glerum — APPROVED** [review]

**Comment by anne-glerum** [comment]
Looks good to me, can I just merge it now?

**Comment by bangerth** [comment]
Yes, please :-)

**Pull request merged**